nf_core/lint/actions_schema_validation.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import logging
+import yaml
+import json
+import jsonschema
+import os
+import glob
+import requests
+
+
+def actions_schema_validation(self):
+    """
+    Validate workflows against a schema
+
+    Uses the JSON Schema from
+    https://json.schemastore.org/github-workflow
+    to verify that all workflow yml files adhere to the correct Schema
+    """
+    passed = []
+    failed = []
+
+    # Only show error messages from schema
+    logging.getLogger("nf_core.schema").setLevel(logging.ERROR)
+
+    # Get all workflow files
+    action_workflows = glob.glob(os.path.join(self.wf_path, ".github/workflows/*.y*ml"))
+
+    # Load the GitHub workflow schema
+    r = requests.get("https://json.schemastore.org/github-workflow", allow_redirects=True)
+    schema = r.json()
+
+    # Validate all workflows against the schema
+    for wf_path in action_workflows:
+        wf = os.path.basename(wf_path)
+
+        # load workflow
+        try:
+            with open(wf_path, "r") as fh:
+                wf_json = yaml.safe_load(fh)
+        except Exception as e:
+            failed.append("Could not parse yaml file: {}".format(wf))
```

Comment:
Might be nice to include `e` in the log to explain what went wrong.. (or a log debug if you want it short?)

Reply:
Yep, was fearing that it get's too messy but can't hurt to have more information. Thanks!